src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+// TODO: remove relative path
+#include "../pooling.hpp"
+#include "arm_compute/runtime/NEON/NEFunctions.h"
+#include "utils/debug_capabilities.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class AclPoolingExecutor : public PoolingExecutor {
+public:
+    AclPoolingExecutor(const ExecutorContext::CPtr context);
+
+    bool init(const PoolingAttrs& poolingAttrs,
+              const std::vector<MemoryDescPtr>& srcDescs,
+              const std::vector<MemoryDescPtr>& dstDescs,
+              const dnnl::primitive_attr &attr) override;
+    void exec(const std::vector<MemoryCPtr>& src,
+              const std::vector<MemoryPtr>& dst,
+              std::unordered_map<int, MemoryPtr> postOpsArgs) override;
+
+    impl_desc_type getImplType() const override {
+        return implType;
+    }
+
+private:
+    PoolingAttrs poolingAttrs;
+    impl_desc_type implType = impl_desc_type::gemm_acl;
```

Comment:
I would propose to put just `impl_desc_type::acl` as long as 'gemm' is specific to matrix multiplication only.

Reply:
Fixed